packages/workflow/src/workflow.ts

```diff
@@ -173,6 +198,79 @@ export function scheduleActivity<R>(
   }) as Promise<R>;
 }
 
+async function startChildWorkflowExecutionNextHandler({
+  options,
+  args,
+  headers,
+  workflowType,
+}: StartChildWorkflowExecutionInput): Promise<[Promise<string>, Promise<unknown>]> {
+  const workflowId = options.workflowId ?? uuid4();
+
+  // TODO: validate options
```

Comment:
TODO needs to be done?

Reply:
I realized there's no special validation required here assuming the user uses typescript.
At a later point I'll add validation for JS users (e.g. passing a number instead of a string).